libImaging/Resample.c

```diff
@@ -5,6 +5,13 @@
 
 #define ROUND_UP(f) ((int) ((f) >= 0.0 ? (f) + 0.5F : (f) - 0.5F))
 
+#ifdef WORDS_BIGENDIAN
+    #define MAKE_UINT32(u0, u1, u2, u3) (u3 | (u2<<8) | (u1<<16) | (u0<<24))
+#else
+    #define MAKE_UINT32(u0, u1, u2, u3) (u0 | (u1<<8) | (u2<<16) | (u3<<24))
+#endif
```

Comment:
LGTM. I've fired up the antique bigendian PPC machine I have here, and the only failures I'm seeing are with 16bit jpeg2k files, but those tests appear to fail all the way back. 

These macros appear to be fine, assuming that we've got enough testing on this part.